src/main/java/org/kainos/ea/resources/JobController.java

```diff
@@ -17,7 +19,9 @@
 @Api("Job Role API")
 @Path("/api")
 public class JobController {
-    private JobService jobService = new JobService();
+    DatabaseConnector databaseConnector = new DatabaseConnector();
+    JobDao jobDao = new JobDao(databaseConnector);
+    private JobService jobService = new JobService(jobDao, databaseConnector);
```

Comment:
This is good practice, but if your job DAO needs the database connector why does it also need to be passed into the job service constructor?